### DISCUSSION TITLE
Filer store migration

### DISCUSSION THREAD

**Discussion by werdnum** [seaweedfs/seaweedfs#4951] [Q&A]
Hi,

My filer is in postgres, which I am finding very slow.

I'm thinking about migrating to Cassandra or even LevelDB, but it's not clear to me how (or whether) I can do an online migration.

What's the best practice right now for doing a migration from one filer store to another?

**Comment by inferno-umar** [comment]
An issue must be filed for this feature

**Comment by wangxf2000** [comment]
I am currently using LevelDB to store the metadata of the filer, and a large number of data files have already been stored. At present, I want to migrate from LevelDB to Redis to meet future expansion needs, but I haven't found a viable solution yet. I would be extremely grateful if anyone could provide some suggestions.

**Reply by chrislusf** [reply]
use this https://github.com/seaweedfs/seaweedfs/wiki/Data-Backup#backup-the-metadata

**Reply by wangxf2000** [reply]
It seems that fs.meta does not support heterogeneous filer stores. I want to migrate from LevelDB to Redis. fs.meta.save works fine, but when I execute fs.meta.load, there is no feedback, and the data is not loaded into the target database.

**Reply by chrislusf** [reply]
it should. How did you run it.

**Reply by wangxf2000** [reply]
Works, very great.
